docs/tutorials/AI Guides/agentkit-flow-guide.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Build Custom AI Agents on Flow with Agentkit
+description: Learn how to configure and deploy AI agents on the Flow testnet using Agentkit, Langchain, and the EVM-compatible Flow environment.
+sidebar_label: Using Agentkit on Flow
+sidebar_position: 1
+---
+
+# Getting Started with Agentkit on Flow
+
+Agentkit is an ecosystem-agnostic modular developer toolkit that lets you rapidly build, deploy, and iterate on AI agents using pre-configured environments and ready-to-use templates.
+
+In this guide, you'll set up your own custom agent running on **Flow’s EVM-compatible testnet**, powered by **Langchain** and **Anthropic’s Claude** LLM.
+
+---
+
+## 🚀 Quickstart
+
+Open your terminal and run:
+
+```bash
+npm create onchain-agent@latest
```

Comment:
How can I get started in an existing project?
I mean manual installation.
`npm install onchain-agent@latest`? or other extra steps?